mozilla/fx-private-relay
# MPP-2581 - Track last call and text times for inbound contacts

For MPP-2581, expand the `InboundContact` model to include new nullable datetime fields, `last_call_time` and `last_text_time`. When sending a text reply, `last_text_time` can be used to determine the last contact that texted, even when the contact called after it texted.

## How to test

Manual testing requires an environment setup for relaying voice calls and text messages. You'll need:

* An account that has phones enabled with a true phone (number 1) and a relay number (number 2)
* A third number (contact 1) that can send and receive texts
* A fourth number (contact 2) that can text and call

The testing starts on the released tag:

* Checkout `2022.11.09`, and run the webserver and any other services needed for phone relay.
  * [x] Send a message from contact 1 to the relay number. The true phone gets the relayed message.
  * [x] Reply from the true phone to the relay number. The reply goes to contact 1.
* Optionally confirm the issue:
   * [x] Send a message from contact 2 to the relay number. The true phone gets the relayed message.
   * [x] Call the relay number from contact 2. The true phone gets a voice call.
   * [x] Reply from the true phone to the relay number. The reply goes to contact 1, not contact 2
* Update to this branch. Run migrations `./manage.py migrate phones`, and restart the server as needed
* Test the changes:
   * [x] Reply from the true phone to the relay number. The reply goes to contact 1.
   * [x] Send a message from contact 2 to the relay number. The true phone gets the relayed message.
   * [x] Call the relay number from contact 2. The true phone gets a voice call.
   * [x] Reply from the true phone to the relay number. The reply goes to contact 2.

## Checklist
